sechub-schedule/src/main/java/com/daimler/sechub/domain/schedule/access/ScheduleUserAccessToProjectValidationService.java

```diff
@@ -33,8 +33,14 @@
 	 * @param projectId
 	 */
 	public void assertUserHasAccessToProject(String projectId) {
+				
 		if (userContextService.isSuperAdmin()) {
-			/* a super admin has always access */
+			/* a super admin has always access to existing projects */
+			
+			if(!accessRepository.hasProjectUserAccess(projectId)) {
+				throw new NotFoundException("Project " + projectId + " does not exist, or you have no access.");
```

Comment:
This happens only when no project access with this project id for any user exists - at schedule domain there is no information available if the project does exist or not.

This happens only for super administerators.
So a message like "Project xyz does not exist, or you have no access"

I prefer:
"Project xyz does not exist, or no user has access at all"

Reply:
done